githooks/prepare-commit-msg

```diff
@@ -115,6 +142,9 @@ $cchar     <what was there before: Previously, ...>\\
 $cchar     <why it needed to change: This was inadequate because ...>\\
 $cchar     <what you did about it: To address this, this patch ...>\\
 $cchar\\
+$cchar     Fixes <GH/Jira issue ID/URL to GH/Jira issue>\\
+$cchar     ---\\
```

Comment:
Can we remove this line? It exists after the `Release note` to indicate the end of the sample commit message body.

Reply:
Done.